Mathlib/Data/Finset/Sigma.lean

```diff
@@ -4,6 +4,7 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Mario Carneiro, Yaël Dillies, Bhavik Mehta
 -/
 import Mathlib.Data.Finset.Lattice
+import Mathlib.Order.CompleteLattice.Finset
 import Mathlib.Data.Set.Sigma
```

Comment:
```suggestion
import Mathlib.Data.Set.Sigma
import Mathlib.Order.CompleteLattice.Finset
```
For future work: why is it imported here?!

Reply:
It's for lemmas like `biSup_finsetSigma`, cf #8964

Reply:
Do we agree that these lemmas should move to `Order.CompleteLattice.Finset`?

Reply:
I'm not opposed to that change - if you want to make a PR doing only that then I can review it now